src/Model/TransformerInterface.php

```diff
@@ -21,7 +21,7 @@
  * @phpstan-type BlockContent array{
  *   id: int|string|null,
  *   name?: string|null,
- *   enabled: boolean,
+ *   enabled: boolean|'1'|'0',
```

Comment:
What is the reason for having `'1'` or `'0'` ?
Can't we always having a bool ?

Reply:
Seems not, or at least not for previous versions of this bundle: https://github.com/sonata-project/SonataPageBundle/issues/1495#issuecomment-1236801143

cc @eerison

Reply:
Yeah, for the previous version many values are string :/

I have the felling that we should fix this snapshot data in 3.x and for users migrate to to 4.x should regenerate wall snapshots, this way we can assume that all data is correct, otherwise we will add too much workaround to make snapshot from 3.x compatible with 4.x code

Reply:
Yeah, if I understand correctly the issue is that the data were not perfect in 3.x ; so we have to add BC-layer now in 4.x.
But we will be able to remove those BC-layer in 5.x.

Should we already add NEXT_MAJOR comment to remove the bc layer and update the phpdoc then ?

Reply:
I will see if I can add some NEXT_MAJOR comment.

Reply:
I tried with a NEXT_MAJOR comment @VincentLanglet